src-electron/generator/helper-endpointconfig.js

```diff
@@ -589,15 +589,16 @@ function endpoint_attribute_min_max_count(options) {
  * @returns All attributes with min max values listed
  */
 function endpoint_attribute_min_max_list(options) {
-  let comment = null
-  let order = options.hash.order
-  if (order == null || order.length == 0) {
-    order = 'def,min,max'
-  }
+  let comment = null;
+  let order = options.hash?.order || 'def,min,max'; // Use optional chaining and default value
```

Comment:
What if `options.hash?.order` is an empty string? I would keep the old code.

Reply:
will do thanks